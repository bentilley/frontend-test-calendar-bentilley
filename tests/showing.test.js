import React from 'react';
import renderer from 'react-test-renderer';

import Showing from 'components/calendar/showing';

describe('Showing component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Showing
                                        time='2.30PM'
                                        minCombined='Â£25.05'
                                      />);
    const instance = component.getInstance();
    const result = instance.render();
    expect(result).toMatchSnapshot();
  });
});
