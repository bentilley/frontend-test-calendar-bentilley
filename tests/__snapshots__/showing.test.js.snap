// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Showing component renders correctly 1`] = `
<li
  className="showing undefined"
  onClick={[Function]}
>
  <h3>
    2.30PM
  </h3>
  <p>
    £
    £25.05
  </p>
</li>
`;
